### DISCUSSION TITLE
Access USB camera connected to Android VIA OTG

### DISCUSSION THREAD

**Discussion by mdjavedakhtar** [dotnet/maui#19917] [Q&A]
Hello
i was trying to show video from a usb webcam connected via OTG to Android phone in MAUI but no success
any help will be appreciated

**Comment by MhAllan** [comment]
afaik, MAUI doesn't have camera view, what are you using?

**Reply by mdjavedakhtar** [reply]
I am aware of it.
I used camara.maui plugin. It shows the internal cameras of Android device but not the usb camera

**Reply by vovanb** [reply]
did you found a solution?

**Reply by bronteq** [reply]
@vovanb  The repo does not seem updated, but you can open an issue there (https://github.com/hjam40/Camera.MAUI)
From Android 10 usb permission is needed, do you have it in the manifest?

**Reply by ghost** [reply]
@mdjavedakhtar just additional info which [CoPilot AI](https://copilot.microsoft.com/shares/RHoj1dRadq13RyABAyZbp) led me to (LENS_FACING_EXTERNAL). [Relevant topic on stackoverflow](https://stackoverflow.com/questions/50802007/communicating-with-the-usb-device-plugged-into-android-phone-for-instance-an-e) and [stackoverflow second topic](https://stackoverflow.com/questions/57846505/accessing-a-usb-camera-using-android-camera2-api).

[Related medium.com blog](https://medium.com/androiddevelopers/camera-enumeration-on-android-9a053b910cb5).

It would be interesting to test how many cameras [Camera.MAUI/Camera.MAUI.ZXing](https://github.com/hjam40/Camera.MAUI) - NumCamerasDetected.

[Technical google group topic](https://groups.google.com/g/android-platform/c/rqj2SKG7Cx4?pli=1).
[Android native code - not really related to MAUI](https://blog.csdn.net/huahua520amy/article/details/132738080).

[This topic is about specific Cognex hardware](https://cmbdn.cognex.com/v2.7.x/knowledge/net-maui/camera-overlay-customization), maybe just for additional info.
